.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: cpp
-dist: trusty
+dist: xenial
 
 compiler:
   - g++
   - clang
 
 before_install:
   - sudo apt-get update -qq
-  - sudo apt-get install -qq nasm g++-multilib gcc-multilib libc6-dev-i386 python3-pip unzip
-  - sudo python3 -m pip install meson==0.44.1
+  - sudo apt-get install -qq nasm g++-multilib gcc-multilib libc6-dev-i386 python3-pip python3-setuptools unzip
```

Comment:
why is this needed ooc?

Reply:
IIRC there was a CI failure without this. I'll check again

Reply:
```
Collecting meson==0.47.0
  Downloading https://files.pythonhosted.org/packages/c0/9b/44cdb8adcbb186be6cba5c93718d0c68f177b0e8082ae00cafa63a1d3535/meson-0.47.0.tar.gz (1.2MB)
    100% |████████████████████████████████| 1.2MB 1.1MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ImportError: No module named 'setuptools'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-dhn678co/meson/
You are using pip version 8.1.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
The command "sudo python3 -m pip install meson==0.47.0" failed and exited with 1 during .
```

Reply:
oic, ok :)